backend/src/main/java/com/roc/app/user/referee/general/RefereeController.java

```diff
@@ -0,0 +1,28 @@
+package com.roc.app.user.referee.general;
+
+import com.roc.app.user.referee.exception.RefereeNotFoundException;
+import com.roc.app.user.referee.general.dto.RefereeCreateRequestDto;
+import com.roc.app.user.referee.general.dto.RefereeResponseDto;
+import jakarta.validation.Valid;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/referee")
```

Comment:
Should be plural form:
```suggestion
@RequestMapping("/api/referees")
```